src/Symfony/Component/HttpFoundation/Session/Storage/Handler/PdoSessionHandler.php

```diff
@@ -687,9 +697,19 @@ private function getMergeSql()
     protected function getConnection()
     {
         if (null === $this->pdo) {
-            $this->connect($this->dsn ?: ini_get('session.save_path'));
+            $this->connect($this->dsn ?: $this->getSessionSavePath());
```

Comment:
-1 for this. there is already a way to configure the DSN without using the ini variable